ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -90,16 +88,22 @@
 {% include 'eos/unsupported-transceiver.j2' %}
 {# tacacs servers #}
 {% include 'eos/tacacs-servers.j2' %}
+!
+{# aaa server groups #}
+{% include 'eos/aaa-server-groups.j2' %}
 {# aaa authentication #}
 {% include 'eos/aaa-authentication.j2' %}
 {# aaa authorization #}
 {% include 'eos/aaa-authorization.j2' %}
 {# aaa accounting #}
 {% include 'eos/aaa-accounting.j2' %}
-{# aaa root #}
-{% include 'eos/aaa-root.j2' %}
+{% if aaa_authentication is arista.avd.defined or aaa_authorization is arista.avd.defined or aaa_accounting is arista.avd.defined %}
```

Comment:
Let's create an `aaa.j2` template that we include and then put all the logic in there. To keep this one clean.

Reply:
Perfect!  I did not want to consolidate templates without blessing from someone on your side.  I will adjust.